selfie-camera/src/main/java/org/odk/collect/selfiecamera/CameraXCamera.kt

```diff
@@ -0,0 +1,137 @@
+package org.odk.collect.selfiecamera
+
+import android.annotation.SuppressLint
+import android.view.View
+import androidx.activity.ComponentActivity
+import androidx.camera.core.CameraSelector
+import androidx.camera.core.ImageCapture
+import androidx.camera.core.ImageCaptureException
+import androidx.camera.core.Preview
+import androidx.camera.lifecycle.ProcessCameraProvider
+import androidx.camera.video.FileOutputOptions
+import androidx.camera.video.Recorder
+import androidx.camera.video.Recording
+import androidx.camera.video.VideoCapture
+import androidx.camera.video.VideoRecordEvent
+import androidx.camera.view.PreviewView
+import androidx.core.content.ContextCompat
+import org.odk.collect.androidshared.livedata.MutableNonNullLiveData
+import org.odk.collect.androidshared.livedata.NonNullLiveData
+import java.io.File
+
+internal class CameraXCamera : Camera {
```

Comment:
Can't it be just `CameraX`? `CameraXCamera` sound weird but maybe only to me :)

Reply:
I'm just following the convention we generally use of `<implementation framework/detail><interface name>`, but I do see your point! I think I'd rather just stick with this at the moment because I can't think of a better name to describe what's going on here.